definitions/tools/pvacseq.wdl

```diff
@@ -113,6 +113,13 @@ task pvacseq {
     File? combined_aggregated_report = "pvacseq_predictions/combined/~{sample_name}.all_epitopes.aggregated.tsv"
     File? combined_filtered_epitopes = "pvacseq_predictions/combined/~{sample_name}.filtered.tsv"
     File? combined_aggregated_metrics_file = "pvacseq_predictions/combined/" + sample_name + ".all_epitopes.aggregated.metrics.json"
+    
+    Array[File?] mhc_i = [mhc_i_all_epitopes, mhc_i_aggregated_report, mhc_i_filtered_epitopes, mhc_i_aggregated_metrics_file] 
+    Array[File?] mhc_ii = [mhc_ii_all_epitopes, mhc_ii_aggregated_report, mhc_ii_filtered_epitopes, mhc_ii_aggregated_metrics_file]
+    Array[File?] combined = [combined_all_epitopes, combined_aggregated_report, combined_filtered_epitopes, combined_aggregated_metrics_file]
+
+    # Don't think this works the way it is intended. Refer to documentation.
+    # https://github.com/openwdl/wdl/blob/main/versions/1.0/SPEC.md#globs
```

Comment:
What is intended and what is happening instead?

Reply:
From the looks of the the command that uses glob, it seemed like it intends to iterate through pvacfuse_predictions/ like this:

| PVACfuse_predictions/
  | mhc_i
    *grabs all files*
  | mhc_ii
    *grabs all files*
  | combined
    *grabs all files*

And output a similar structure to the above.

However the the real result of that was like this:

| PVACfuse_predictions/
   | mhc_i
     *grabs all files*

Leading to the output:

PVACfuse_predictions/<content of mhc_i>

And the documentation does state that glob skips folders and their contents. Only thing that is slightly vague to me is the use of the double * and why did it pick mhc_i as opposed to another folder unless that was just purely due to order.